BWWalkthroughExampleTests/BWWalkthroughExampleTests.swift

```diff
@@ -1,36 +0,0 @@
-//
```

Comment:
Ah, this was just the empty test file. It's removed (it's visible in the changelog). Better just revert that change (in case upstream for instance adds tests), it should be symptomless from your point of view. Remember that you don't need to refer to the file as we directly import the file (for now before we can create real frameworks after dropping iOS7 support).